eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -165,13 +166,20 @@ private function cacheClear(OutputInterface $output)
      * IMPORTANT: This is done using a command because config has change, so container and all services are different.
      *
      * @param OutputInterface $output
+     * @param null|string $siteaccess
      */
-    private function indexData(OutputInterface $output)
+    private function indexData(OutputInterface $output, string $siteaccess = null)
```

Comment:
`string` type hint is available since PHP 7.0+ but we require `"php": "~5.6|~7.0"` in `composer.json`